software/apps/Makefile

```diff
@@ -24,9 +24,14 @@ else
 	ALL := $(filter-out systolic/%,$(APPS))
 endif
 
+ALL_LLVM := $(filter-out synth, $(ALL))
+
```

Comment:
This is because we don't have the extension in LLVM yet, right? Once we fix this, we can remove this distinction again?

Reply:
Yes, you are right, I add these just for the CI checking on ASM codes.